src/indexing.jl

```diff
@@ -95,7 +95,7 @@ end
     end
 end
 
-# MAYBE: fixed-size indexing with bools? They would have to be Val's...
+# MAYBE: fixed-wx indexing with bools? They would have to be Val's...
```

Comment:
A typo. I thought I had force pushed to change that back. Let me check my branch.